x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/columns/selected_filters.test.tsx

```diff
@@ -40,7 +40,9 @@ describe('SelectedFilters', function () {
 
     await waitFor(() => {
       screen.getByText('Chrome');
-      screen.getByTitle('Filter: Browser family: Chrome. Select for more filter actions.');
+      screen.getByTestId(
+        'filter filter-enabled filter-key-Browser family filter-value-Chrome filter-unpinned'
```

Comment:
Had to get rid of the `title` attribute for the filters in favour of using an `EuiTooltip`, so had to grab by `data-test-subj` instead.